rpc/rpc-interfaces.go

```diff
@@ -9,14 +9,14 @@ import (
 	"time"
 )
 
-// RPCClient describes the functions an RPC Client performs
-type RPCClient interface {
+// client describes the functions an RPC Client performs
+type client interface {
```

Comment:
Why were these made private? They are used in the public API in NewRegistrationAuthorityServer, etc.